exec/tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = {py36,py37,py38}-beam{25,26,27},docs,manifest,check-formatting,lint
+envlist = {py36,py37,py38}-beam{25,26,27,28},docs,manifest,check-formatting,lint
```

Comment:
I believe our policy is only supporting 3 versions, so we should drop 25, right @econchick ?

Reply:
Yes, that's true. Drop `25` here and the entire line 15 (for this tox and the other)